### PR TITLE
Some fixes

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by nyuszika7h** [comment]
testing commit announcing

File: irc-against-humanity.json [hunk 1]
```diff
@@ -19,44 +19,45 @@
...
 	{"text": "%s and %s. I ship it.", "pick": 2},
-	{"text": "%s looked through my browser history, only to discover %s.", "pick": 2}
+	{"text": "%s looked through my browser history, only to discover %s.", "pick": 2},
+	{"text": "How to beat your children without leaving any sign of injury"}
 ],
 "answers": [
-	"Imaginary boyfriend",
+	"An imaginary boyfriend",
 	"Frolicking in a meadow",
 	"Your own mother",
```
Comment (ghost) [inline irc-against-humanity.json:27]: Maybe this should be just "Your mom".

File: irc-against-humanity.json [hunk 2]
```diff
@@ -19,44 +19,45 @@
...
 	"Faking a smile for a family portrait",
-	"Sexually harassing a co-worker and then finding out she's your cousin",
+	"Sexually harassing a co-worker then finding out she's your cousin",
 	"Used panties",
 	"Violently fisting until your knuckles are bruised",
-	"IRC bot",
+	"An IRC bot",
 	"Premature ejaculation",
-	"A fart that almost wan't a fart",
+	"A fart that almost wasn't a fart",
```
Comment (nyuszika7h) [inline irc-against-humanity.json]: There's already a similar card, do we really need this?

"When you fart and a little bit comes out"

File: irc-against-humanity.json (hunk 1 above)
Comment (melamity) [inline irc-against-humanity.json:27]: noita666 gave me it as "Your mom" but I thought "Your mother" fit, idk why it's NOW "Your own mother", unless I did that by mistake. noita666 seemed pretty happy for it to be "Your mother", either way.

**Comment by ghost** [comment]
I'm going to merge this. If someone still has a problem with it, we can discuss it later.

**Pull request merged**